_episodes/01-software-development-intro.md

```diff
@@ -16,18 +16,15 @@ keypoints:
 how the application itself interacts with its users."
 - "MVC is a software design architecture which divides the application into three interconnected modules: Model (data), 
 View (user interface), and Controller (input/output and data manipulation)."
-- "Software project template to be used throughout this workshop is an example of an MVC application that manipulates 
+- "The software project template we use throughout this workshop is an example of an MVC application that manipulates 
 patients’ inflammation data and performs basic statistical analysis using Python."
 ---
 ## Introduction
-One of the first things to consider in software development is the design architecture of a software project. 
-Hence, we start by looking into design architectures and, 
-more specifically, the Model-View-Control design architecture used by the example software project we will be developing 
-over the course of this workshop. There are other things to consider early on in software development - for example, 
-[choosing a license](/32-preparing-software-reuse/index.html#choosing-an-open-source-licence) for your 
-project or [writing documentation](/32-preparing-software-reuse/index.html#documenting-code-to-improve-reusability). 
-For the purposes of this workshop we cover them later, but be aware that in the normal run of the things they should 
-addressed early.
+One of the big differences between novice and intermediate software development is planning the structure of your code. 
+A novice will often make up the structure of their code as they go along - but for more advanced software development, 
+we need to plan this structure beforehand. There is a number of ways to structure code called *software design architectures*. 
```

Comment:
It might read better to say "...this structure beforehand, which we call a *software architecture*'. I think the way this is phrased presupposes that there are definitive set architectures for everything, which there aren't